example/pubspec.yaml

```diff
@@ -6,14 +6,15 @@ publish_to: 'none'
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=1.17.0 <2.0.0"
```

Comment:
I think this range is incorrect right? Now with 2.0 on stable we can't use < 2 anymore. Maybe we should only target >= 2 from now on?